contracts/src/core/EigenDADirectory.sol

```diff
@@ -118,91 +118,91 @@ contract EigenDADirectory is IEigenDADirectory, IEigenDASemVer {
-        return ConfigRegistryLib.getActivationKeyBytes(nameDigest, index);
+    function getActivationKeyTimeStamp(bytes32 nameDigest, uint256 index) external view returns (uint256) {
+        return ConfigRegistryLib.getActivationKeyTimeStamp(nameDigest, index);
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getCheckpointBytes32(bytes32 nameDigest, uint256 index)
+    function getCheckpointBlockNumber(bytes32 nameDigest, uint256 index)
         external
         view
-        returns (ConfigRegistryTypes.Bytes32Checkpoint memory)
+        returns (ConfigRegistryTypes.BlockNumberCheckpoint memory)
     {
-        return ConfigRegistryLib.getCheckpointBytes32(nameDigest, index);
+        return ConfigRegistryLib.getCheckpointBlockNumber(nameDigest, index);
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getCheckpointBytes(bytes32 nameDigest, uint256 index)
+    function getCheckpointTimeStamp(bytes32 nameDigest, uint256 index)
         external
         view
-        returns (ConfigRegistryTypes.BytesCheckpoint memory)
+        returns (ConfigRegistryTypes.TimeStampCheckpoint memory)
     {
-        return ConfigRegistryLib.getCheckpointBytes(nameDigest, index);
+        return ConfigRegistryLib.getCheckpointTimeStamp(nameDigest, index);
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getConfigNameBytes32(bytes32 nameDigest) external view returns (string memory) {
-        return ConfigRegistryLib.getNameBytes32(nameDigest);
+    function getConfigNameBlockNumber(bytes32 nameDigest) external view returns (string memory) {
+        return ConfigRegistryLib.getNameBlockNumber(nameDigest);
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getConfigNameBytes(bytes32 nameDigest) external view returns (string memory) {
-        return ConfigRegistryLib.getNameBytes(nameDigest);
+    function getConfigNameTimeStamp(bytes32 nameDigest) external view returns (string memory) {
+        return ConfigRegistryLib.getNameTimeStamp(nameDigest);
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getAllConfigNamesBytes32() external view returns (string[] memory) {
-        return ConfigRegistryLib.getNameListBytes32();
+    function getAllConfigNamesBlockNumber() external view returns (string[] memory) {
+        return ConfigRegistryLib.getNameListBlockNumber();
     }
 
     /// @inheritdoc IEigenDAConfigRegistry
-    function getAllConfigNamesBytes() external view returns (string[] memory) {
-        return ConfigRegistryLib.getNameListBytes();
+    function getAllConfigNamesTimeStamp() external view returns (string[] memory) {
+        return ConfigRegistryLib.getNameListTimeStamp();
     }
 
     /// @inheritdoc IEigenDASemVer
     function semver() external pure returns (uint8 major, uint8 minor, uint8 patch) {
         major = 1;
-        minor = 1;
+        minor = 2;
```

Comment:
shouldnt it be 2.0.0? this is a big breaking change, all function names are changing.

Reply:
yeah makes sense - was bumping minor since this was deployed prematurely on testnets